README.md

```diff
@@ -1 +1,45 @@
-# create_engineers_website
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# creative engineers website:  mobile and desktop version 
+> building the first 2 pages (Home and About pages) of a fantasy company that i dreamed to establish before  for mobile and Dektop devices.
+that was done by choosing [This Template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) that is actually designed by this creative graphic desginer Guy  [Cindy Shin in Behance](https://www.behance.net/adagio07).
+
+
+## Built With
+
+- html , css and javascript
+
+
+## Live Demo 
+
+[Live Demo Link](https://mohamed20163858.github.io/create_engineers_website/)
+
```

Comment:
your project does not have a video representation. please add this and try again 😸

Reply:
@alexander16108 sorry i thought it will be added seperately